package.json

```
@@ -4,8 +4,8 @@
   "description": "Projeto criando um CRUD com qualidade!",
   "main": "index.js",
   "scripts": {
-    "start": "node ./core/crud.js",
-    "dev": "nodemon ./core/crud.js",
+    "start:crud": "npx ts-node ./core/crud.ts",
+    "dev:crud": "nodemon --ext ts,tsx --exec 'npm run start:crud'",
```

No windows, você pode precisar deixar o script assim: 
```suggestion
    "dev:crud": "nodemon --ext ts,tsx --exec \"npm run start:crud\"",
```

---

Acredito que seria interessante deixar isso documentado também no link da aula.